src/main/java/stsjorbsmod/patches/WitheringPatch.java

```diff
@@ -0,0 +1,48 @@
+package stsjorbsmod.patches;
+
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.IntangiblePlayerPower;
+import javassist.CtBehavior;
+import stsjorbsmod.cards.cull.Withering;
+import stsjorbsmod.powers.WitherPower;
+
+public class WitheringPatch {
+    @SpirePatch(clz = IntangiblePlayerPower.class, method = "atDamageFinalReceive")
+    public static class ReduceIntangibleDamageReductionPatch
+    {
+        @SpirePrefixPatch
+        public static SpireReturn patch(IntangiblePlayerPower __instance, float dmg, DamageInfo.DamageType type) {
+            if(AbstractDungeon.player.hasPower(WitherPower.POWER_ID) && dmg > 1)
+            {
+               return SpireReturn.Return(2.0f);
+            }
+            return SpireReturn.Continue();
+        }
+    }
+    //WHY IS INTANGIBLE HARDCODED IN THE ABSTRACT PLAYER CLASS POEIAÖRL KELKR ÖLÖLREK ÖOLKE
```

Comment:
It's actually worse than you're imagining; it's technically possible for monsters to have IntangiblePlayerPower in certain weird circumstances, and since our mod has some cases where debuffs can be reflected from the player onto monsters, it would probably be safest for `WitherPower` to apply to them too, so we'll want a similar patch in `AbstractMonster::damage`